tibco_ems/manifest.json

```diff
@@ -0,0 +1,58 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "32445b00-582f-4e56-9c4d-87944d5c347b",
+  "app_id": "tibco-ems",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "The Tibco EMS integration",
+    "title": "tibco_ems",
+    "media": [],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS",
+      "Submitted Data Type::Metrics"
```

Comment:
Please add an `Offering::Integration` tag and 1+ `Category` classifier tag (see [list](https://github.com/DataDog/dd-source/blob/main/domains/assets/shared/libs/python/assetlib/classifiers.py#L25))